README.md

```diff
@@ -120,6 +120,34 @@ To clean up Kubernetes objects created by Sonobuoy, run:
 sonobuoy delete
 ```
 
+### Custom registries and air-gapped testing
+
+In air-gapped deployments where there is no access to the public Docker registries
+Sonobuoy supports running end-to-end tests with custom registries. This enables
+you to test your air-gapped deployment once you've loaded the necessary images
+into a registry that is reachable by your cluster.
+
+Just provide the `--e2e-repo-list` parameter and pass it the path to a local list
```

Comment:
nit: "pass it the path to a local list of registries". This sounds like I can pass `--e2e-repo-list gcr.io/heptio-images,docker.io`.

Reply:
```
Just provide the `--e2e-repo-config` parameter and pass it the path to a local
yaml file pointing to the registries you'd like to use.
```